src/main.ts

```diff
@@ -6,13 +6,34 @@ import { handleGenerateAddress } from './handlers/gen.address.handler';
 import { networkSelectorHandler } from './handlers/network.selector.handler';
 import { handleTransactions } from './handlers/transactions.handler';
 
+import { TransactionService } from './services/transaction.service';
+
 // Define CLI commands
 program
     .command('start')
     .description('Start the interactive terminal')
     .action(async () => {
         const user = await handleAuth();
         const { appNetwork } = await networkSelectorHandler();
+
+        /**
+         * Below is an example of how to use the TransactionService to create a signed transaction
+         * you can use the credentials test:test123456789012 to run this example
+         */
+        const addressWithBalance = '2MwrinsNPrqwqup5G5DQ5BG2XnxjLs8y28z';
+        const redeemScriptHex = TransactionService.createRedeemScript('Btrust Builders');
```

Comment:
you cannot create a redeem script from string.

Reply:
Yes.
I have separated the code for better testing. here https://github.com/IdrisAkintobi/bitcoin-transaction